app/main.py

```diff
@@ -2,0 +1,8 @@
+def copy_file(command: str) -> None:
+    check_list = command.split(" ")
+    if check_list[0] != "cp" or check_list[1] == check_list[2]:
+        return None
+
+    with (open(check_list[1], "r")
+          as file_in, open(check_list[2], "w") as file_out):
```

Comment:
The `open` statement is missing a closing parenthesis. Ensure that the syntax is correct to avoid a `SyntaxError`.